charts/oes/templates/ops-svc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
should this file be called ops-svc or opa-svc? should be conditional

Reply:
Done.